TweetBook/Controllers/V1/PostsController.cs

```diff
@@ -21,5 +23,25 @@ public IActionResult GetAll()
         {
             return Ok(_posts);
         }
+
+        [HttpPost(ApiRoutes.Posts.Create)]
+        public IActionResult Create([FromBody] CreatePostRequest postRequest)
+        {
+
+            var post = new Post { Id = postRequest.Id };
+
+            if(string.IsNullOrEmpty(post.Id))
+                post.Id = Guid.NewGuid().ToString();
+            
+            _posts.Add(post);
+
+            var baseUrl = $"{HttpContext.Request.Scheme}://{HttpContext.Request.Host.ToUriComponent()}";
+
+            var locationUri = baseUrl + "/" + ApiRoutes.Posts.Get.Replace("{postId}", post.Id);
+
+            var response = new PostResponse { Id = post.Id };
```

Comment:
Where is this variable used afterwards?

Reply:
Also, can I see a screenshot of the response for this endpoint?

Reply:
![image](https://user-images.githubusercontent.com/74786468/155324252-36d721fa-05ab-4030-bb76-3bb0e81f05d2.png)

Reply:
> Where is this variable used afterwards?

I'm not using yet anywhere, i can get rid of it for now, thanks.